.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,39 @@
 <!--
-Thanks for sending a pull request!
+Thank you for contributing! Please remember to:
 
-Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests).
+- Read our contributing guidelines: https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests
+- Sign your commits with your real name. You can amend previous commits with `git commit -s --amend`.
+-->
+
+#### Description
+<!-- Provide a clear summary of your changes and why they are needed. -->
 
-Finally, be sure to sign commits with your real name. Since by opening
-a PR you already have commits, you can add signatures if needed with
-something like `git commit -s --amend`.
+Fixes:
+<!--
+Link to the issue(s) this Pull Request addresses.
+- If it fixes a single issue, use:
+    Fixes: #00000
+    Fixes: https://github.com/containers/common/issues/00000
+    Fixes: https://issues.redhat.com/browse/RHEL-00000
+    Fixes: RHEL-00000
+- If it fixes multiple issues, use commas:
+    Fixes: #00000, #00001, #0002
+    Fixes: https://github.com/containers/common/issues/00000, https://github.com/containers/common/issues/00001
+    Fixes: https://issues.redhat.com/browse/RHEL-00000, https://issues.redhat.com/browse/RHEL-00001
+    Fixes: RHEL-00000, RHEL-00001
+- If there is no related issue, you can leave this section blank.
 -->
 
-#### Does this PR introduce a user-facing change?
+---
 
+#### Does this PR introduce a user-facing change?
 <!--
-If no, just write `None` in the release-note block below. If yes, a release note
-is required: Enter your extended release note in the block below. If the PR
-requires additional action from users switching to the new release, include the
-string "action required".
+- If no, enter "None" in the `release-note` block below.
+- If yes, enter a release note.
+- If the change requires user action, add "action required" to the release note.
 
-For more information on release notes, please follow the Kubernetes model:
-https://git.k8s.io/community/contributors/guide/release-notes.md
+For details on writing release notes, see: https://git.k8s.io/community/contributors/guide/release-notes.md
 -->
-
 ```release-note
-
+None
```

Comment:
I'm not sure if it's a good idea to set the default value to `None`.  Many people forget to include a release note, which causes CI to overlook some changes worth mentioning in the release notes. WDYT? @Luap99 , @mheon

Reply:
I agree.  If we set to none, that will be the last meaningful release note helper we get.